components/server/src/iam/iam-session-app.ts

```diff
@@ -108,6 +108,13 @@ export class IamSessionApp {
         await this.teamDb.addMemberToTeam(user.id, organizationId);
         if (firstMember) {
             await this.teamDb.setTeamMemberRole(user.id, organizationId, "owner");
+
+            // remove the admin-user from the org
+            try {
+                await this.teamDb.removeMemberFromTeam(BUILTIN_INSTLLATION_ADMIN_USER_ID, organizationId);
+            } catch {
+                // ignore errors if membership would not exist.
+            }
```

Comment:
If we land this, and you mess up your SSO config, you will no longer be able to edit the SSO config on the org, because, well, you're not in the org.

I believe the right solution until we have better perms model is to not show the user - filter it out of the results

Reply:
> If we land this, and you mess up your SSO config, you will no longer be able to edit the SSO config

That's not the case. 

If you're at this point, your real user account has been created, *after* you successfully went through the entire flow, and you are already owner of the Org. 

The worst thing can happen is, that you're not receiving the cookies, but the cell's setup is completed. In that case, you're not auth'd, thus you will see the Login page. If you continue with SSO, you will be logged in.

Reply:
Yes, but if you/someone else deregister the SSO provider on the provider side, you have no way of updating the config in gitpod.

Reply:
We can extend the admin login part, if you think the "rescue mode" is relevant.